components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -595,12 +783,12 @@ subroutine crm_physics_tend(ztodt, state, tend, ptend, pbuf, cam_in, cam_out, &
    ! call pbuf_set_field(pbuf, pbuf_get_index('PRAIN' ), 0.0_r8 )
    ! call pbuf_set_field(pbuf, pbuf_get_index('NEVAPR'), 0.0_r8 )
 
-   !!! set convective rain to be zero for PRAIN already includes precipitation production from convection. 
-   ! call pbuf_set_field(pbuf, pbuf_get_index('RPRDTOT'), 0.0_r8 )
-   ! call pbuf_set_field(pbuf, pbuf_get_index('RPRDDP' ), 0.0_r8 )
-   ! call pbuf_set_field(pbuf, pbuf_get_index('RPRDSH' ), 0.0_r8 )
-   ! call pbuf_set_field(pbuf, pbuf_get_index('ICWMRDP'), 0.0_r8 )
-   ! call pbuf_set_field(pbuf, pbuf_get_index('ICWMRSH'), 0.0_r8 )
+   ! set convective rain to be zero for PRAIN already includes precipitation production from convection. 
```

Comment:
Shouldn't these go away if offline physics is disabled?

Reply:
Yea, but as I commented elsewhere, removing them causes problems that would require modifying other files outside of the crm folder. It will be way easier to do that later in a dedicated PR so the non-MMF folks don't have to wade through a huge amount of changes (like I'm making you do with this PR).